pki.spec

```diff
@@ -49,7 +49,13 @@ Source: https://github.com/dogtagpki/pki/archive/v%{version}%{?_phase}/pki-%{ver
 # Java
 ################################################################################
 
+%if 0%{?fedora} && 0%{?fedora} >= 33
+%define java_home %{_usr}/lib/jvm/java-11-openjdk
+%define min_java_version 1:11
```

Comment:
Our CI runs on `F32` and `F31`. The logic here won't get executed in our CI (ie) our CI still will install Java 8. Did you get a chance to test this PR on a rawhide build? (pki-core was broken in rawhide last week when I tried to build beta version)

From CI:
````
java-1.8.0-openjdk                x86_64  1:1.8.0.252.b09-1.fc32                  updates                                          281 k
java-1.8.0-openjdk-headless       x86_64  1:1.8.0.252.b09-1.fc32                  updates                                           32 M
````